app/pods/print-overviews/decisions/agendaitems/template.hbs

```diff
@@ -34,6 +34,7 @@
         {{#each (await agendaitemRow.treatments) as |treatment|}}
           <Agenda::Agendaitem::AgendaitemDecision
             @subcase={{await agendaitemRow.content.agendaActivity.subcase}}
+            @agendaitemIsConfidential={{await agendaitemRow.content.agendaActivity.subcase.confidential}}
```

Comment:
since we are using `agendaitemRow.content.agendaActivity.subcase` twice now, could we extract it to a `{{#let}}`?
https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/8a4b848fcebcef9c751e0ced9311e4a70f2d66f6/app/components/agenda/agenda-header/agenda-action-popup-agendaitems.hbs#L18

Just a suggestion, whatever works is fine.
I can live with this current code since this route is mostly unused by the users I think.